desc/optimize/utils.py

```diff
@@ -292,3 +292,15 @@ def evaluate_quadratic(J, g, s, diag=None):
     l = jnp.dot(s, g)
 
     return 0.5 * q + l
+
+
+def find_matching_inds(arr1, arr2):
+    """Find indices into arr2 that match rows of arr1."""
+    arr1, arr2 = map(np.atleast_2d, (arr1, arr2))
+    inds = []
+    for i, a in enumerate(arr2):
+        x = np.isclose(arr1, a, rtol=1e-14, atol=1e-14)
+        j = np.where(x.all(axis=1))[0]
+        if len(j):
+            inds.append(i)
+    return np.asarray(inds)
```

Comment:
I get errors when I try to run the following:
```a = np.array([0, 2, 5, 9, 3, 4, 8])
b = np.array([-4, -5, 0, 1, 3])
inds = find_matching_inds(a, b)
```

And in the doc string, does it return the indices of arr2 or arr1?

Reply:
updated the docstring to make it clearer. The two arrays should have the same number of columns, and the indices are into `arr2`